gstreamer/src/message.rs

```diff
@@ -1183,13 +1183,17 @@ macro_rules! message_builder_generic_impl {
                     ffi::gst_message_set_seqnum(msg, seqnum.to_glib());
                 }
 
-                {
-                    let s = StructureRef::from_glib_borrow_mut(
-                        ffi::gst_message_get_structure(msg) as *mut _
-                    );
-
-                    for (k, v) in self.other_fields {
-                        s.set_value(k, v.to_send_value());
+                if !self.other_fields.is_empty() {
+                    // TODO: add call ffi::gst_message_writable_structure instead
+                    // when https://bugzilla.gnome.org/show_bug.cgi?id=792928 is merged
+                    // and matching feature is activated
```

Comment:
For the 0.11 release, we should make the other-fields API conditional on the `v1_14` feature (and not have it at all for older versions). But that's for when 1.14 is actually release, in 1-2 months or so